presto-hive/src/main/java/com/facebook/presto/hive/HivePageSink.java

```diff
@@ -283,7 +284,7 @@ private HiveRecordWriter createWriter(List<Object> partitionRow)
         checkArgument(partitionRow.size() == partitionColumnNames.size(), "size of partitionRow is different from partitionColumnNames");
 
         List<String> partitionValues = partitionRow.stream()
-                .map(Object::toString) // todo this seems wrong
+                .map(Objects::toString)
```

Comment:
@dain I couldn't find a way to represent hive's NULL as a string in Presto. If it should be done as @yuananf suggested, I can change it to
~~~
.map(partitionValue -> partitionValue == null ? 
               HIVE_DEFAULT_DYNAMIC_PARTITION : partitionValue.toString())
~~~

However it does not seem right to me since we are dealing with values not keys.